spec/prog/vm/nexus_spec.rb

```diff
@@ -227,6 +227,8 @@ def requested_disk_size(st)
       expect(vm).to receive(:nics).and_return([nic]).at_least(:once)
       expect(vm).to receive(:cloud_hypervisor_cpu_topology).and_return(Vm::CloudHypervisorCpuTopo.new(1, 1, 1, 1))
       expect(vm).to receive(:pci_devices).and_return([pci]).at_least(:once)
+      prj.set_ff_vm_public_ssh_keys(["operator_ssh_key"])
```

Comment:
The feature flag introduced in the next commit

<img width="932" alt="Screenshot 2024-07-16 at 12 11 33" src="https://github.com/user-attachments/assets/e2f3fac1-3022-4a91-8601-0c538ad8661b">

Reply:
Moved the test changes to next commit.